addon/mixins/editable.js

```diff
@@ -6,9 +6,10 @@ export default Mixin.create({
   classNameBindings: [`editStatus.status`],
 
   editStatus: computed('manager.editStatus', 'column.key', function() {
-    let { item, key } = this.manager.editStatus || {}
-    if (this.item === item && this.column.key === key) {
-      return this.manager.editStatus;
+    let editStatus = this.manager.get('editStatus').filterBy('id', this.element.id)
+
+    if (editStatus[0] && editStatus[0].id === this.element.id) {
```

Comment:
I'm pretty sure `editStatus[0].id === this.element.id` is always true since you use it for the `filterBy`. And instead of use a `filterBy` just use a `find` which will return you the first match (what you want)